src/client/mod.rs

```diff
@@ -62,6 +62,7 @@ use std::fmt;
 use std::mem;
 use std::sync::Arc;
 use std::time::Duration;
+use std::task::Context;
```

Comment:
`task` is imported below, so `poll_ready` can just use `&mut task::Context<'_>`.

Reply:
Fixed!